aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -435,7 +436,7 @@ impl AptosVM {
                         gas_meter,
                     )?;
                 } else {
-                    return Err(PartialVMError::new(StatusCode::VERIFICATION_ERROR)
```

Comment:
I think the original reason of using this error is to discard the results and cause a new session to create, which will clean the published modules in the previous session.

Reply:
Ah ... this makes sense at this time. Good that we caught it.